sources/packages/backend/apps/api/src/services/student-file/student-file.service.ts

```diff
@@ -72,6 +73,10 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
     newFile.student = { id: studentId } as Student;
     newFile.creator = { id: auditUserId } as User;
     newFile.virusScanStatus = VirusScanStatus.InProgress;
+    // Generate the file hash.
+    newFile.fileHash = createHash("sha256")
+      .update(createFile.fileContent)
```

Comment:
This could a generic operation which can be put in a util method.

Reply:
It a single method call that I did not evaluate as complex enough, so I did not put it in a helper for now.
When I execute the upcoming hashes, I may reconsider it, but I would like to move on with it for this PR, unless it is a blocker.